app/Http/Controllers/ClientPortal/PaymentMethodController.php

```diff
@@ -106,6 +106,26 @@ public function update(Request $request, $id)
         //
     }
 
+    public function verify(ClientGatewayToken $payment_method)
+    {
+        $gateway = auth()->user()->client->getCreditCardGateway();
+
+        return $gateway
+            ->driver(auth()->user()->client)
+            ->setPaymentMethod('App\\PaymentDrivers\\Stripe\\ACH')
```

Comment:
we should change this to resolve ACH::class rather than inject the string like this.

Reply:
Yeh, you right. This is just placeholder until I get proper selecting method.